src/k8s/variables.tf

```diff
@@ -234,6 +234,26 @@ variable "autoscaling_specs" {
   ))
 }
 
+variable "fa_autoscaling_specs" {
+  type = map(object({
+    namespace    = string
```

Comment:
@antoniocapizzi95 Namespace is not used in resource definition, this field should be removed